src/app/dashboard/components/Dashboard.jsx

```diff
@@ -0,0 +1,14 @@
+'use client'
+
+// import Overview from '../overview/components/Overview'
+import SidebarDashboard from './sidebar/Sidebar'
+
+function Dashboard () {
+  return (
+    <div className='flex h-full'>
+      <SidebarDashboard />
+      {/* <Overview /> */}
```

Comment:
> when clicking on dashboard from the nav bar I think it should render by default the overview,

Indeed this is the expected behaviour, it would be best to have it this way.

Reply:
Oki, I implemented this in the next Pr in contracting v.0. if you think this is enough only for the overview, let me know so we can merge